mujoco_warp/_src/test_util.py

```diff
@@ -217,6 +217,7 @@ def benchmark(
     list: Number of contacts.
     list: Number of constraints.
     list: Number of solver iterations.
+    int: Number of converged worlds.
```

Comment:
don't forget to change the function annotation!  need to add one more list to `Tuple`

(it might be time to just declare a struct but we can do that later)